TriblerGUI/widgets/tablecontentdelegate.py

```diff
@@ -169,6 +169,28 @@ def draw_subscribed_control(self, painter, option, index, data_item):
         return True
 
 
+class RatingControlMixin(object):
+    def draw_rating_control(self, painter, option, index, data_item):
+        # Draw empty cell as the background
+        self.paint_empty_background(painter, option)
+
+        if u'type' in data_item and data_item[u'type'] != u'channel':
+            return True
+        if data_item[u'status'] == 1000:  # LEGACY ENTRIES!
+            return True
+        if data_item[u'state'] == u'Personal':
```

Comment:
I guess we should still paint the rating for a personal channel. Users are typically curious about how their channel fares :wink: However, the clicking event should be disabled for it to prevent user unsubscribing from their own channel...